### PR TITLE
Upgrade lighthouse and add new metrics

### LINKED ISSUE #5624: Bump lighthouse from 5.6.0 to 6.1.0
In order to update the new metrics that got introduced in Lighthouse v6 we should update our dependency in the `@wdio/devtools-service`. There might be some work required to ensure that our Lighthouse integration continues to work as some file paths might have changed.

### PR DESCRIPTION
**Pull request by SrinivasanTarget**
## Proposed changes

Fix #5624 Lighthouse has evolved a-lot. PR contains:

- [x] Upgrade to the latest version of the lighthouse
- [x] getMetrics will also cover new metrics introduced in the lighthouse

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

File: packages/wdio-devtools-service/package.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
     "@wdio/logger": "6.0.16",
     "chrome-remote-interface": "^0.28.0",
     "core-js": "~3.6.0",
-    "lighthouse": "^5.1.0",
+    "lighthouse": "^6.1.0",
```
Comment (christian-bromann) [inline packages/wdio-devtools-service/package.json:41]: updating to v6 of Lighthouse is likely to break at a lot of places where we use internal Lighthouse interfaces to calculate the metrics.

Comment (SrinivasanTarget) [reply packages/wdio-devtools-service/package.json:41]: @christian-bromann I have updated the breaking changes and everything works fine locally. Feel free to review it now.

File: packages/wdio-devtools-service/src/auditor.js [hunk 2]
```diff
@@ -94,22 +94,25 @@ export default class Auditor {
     }
 
     async getMetrics () {
-        const timeToFirstByte = await this._audit(TTFBMetric, { URL: this.url })
+        const serverResponseTime = await this._audit(ServerResponseTime, { URL: this.url })
         const result = await this._audit(Metrics)
         const metrics = result.details.items[0] || {}
         return {
             estimatedInputLatency: metrics.estimatedInputLatency,
-            timeToFirstByte: Math.round(timeToFirstByte.numericValue, 10),
```
Comment (christian-bromann) [inline packages/wdio-devtools-service/src/auditor.js:102]: Can we keep the `timeToFirstByte` to stay backwards compatible? Let's add a comment to let people know why we have this value twice.

Comment (SrinivasanTarget) [reply packages/wdio-devtools-service/src/auditor.js:102]: Sure, fixed.

File: packages/wdio-devtools-service/src/auditor.js [hunk 3]
```diff
@@ -94,22 +94,26 @@ export default class Auditor {
...
+            timeToFirstByte: Math.round(serverResponseTime.numericValue, 10),
```
Comment (christian-bromann) [inline packages/wdio-devtools-service/src/auditor.js:102]:
```suggestion
            /**
             * keeping TTFB for backwards compatibility
             */
            timeToFirstByte: Math.round(serverResponseTime.numericValue, 10),
```

Comment (SrinivasanTarget) [reply packages/wdio-devtools-service/src/auditor.js:105]: I slept yesterday. I see you have already made the change. Thanks a lot.

**Pull request merged**